components/map/MapPointBottomSheet.tsx

```diff
@@ -0,0 +1,141 @@
+import BottomSheet, { BottomSheetFooterProps, BottomSheetScrollView } from '@gorhom/bottom-sheet'
+import { forwardRef, Ref, useCallback, useEffect, useMemo, useRef, useState } from 'react'
+import { LayoutAnimation, View } from 'react-native'
+
+import NavigateBottomSheetFooter from '@/components/map/NavigateBottomSheetFooter'
+import Divider from '@/components/shared/Divider'
+import Field from '@/components/shared/Field'
+import Icon from '@/components/shared/Icon'
+import PressableStyled from '@/components/shared/PressableStyled'
+import Typography from '@/components/shared/Typography'
+import { useTranslation } from '@/hooks/useTranslation'
+import { SelectedPoint } from '@/modules/map/types'
+import { normalizePoint } from '@/modules/map/utils/normalizePoint'
+
+type Props = {
+  point: SelectedPoint
+}
+
+const EXCLUDED_ATTRIBUTES = new Set(['address', 'name', 'navigation', 'kind', 'id'])
+
+const MapPointBottomSheet = forwardRef<BottomSheet, Props>(({ point }, ref) => {
+  const t = useTranslation('MapScreen.PointBottomSheet')
+  const [index, setIndex] = useState(-1)
+  const [footerHeight, setFooterHeight] = useState(0)
+
+  const np = useMemo(() => normalizePoint(point), [point])
```

Comment:
Please don't use abbreviations in variable names, use more descriptive variable name than `np`.

We would usually name it `formattedMapPoint`.